engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/util/MemoryUtil.java

```diff
@@ -0,0 +1,51 @@
+package org.enso.interpreter.arrow.util;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.ByteBuffer;
+
+public class MemoryUtil {
+
+  private static Constructor<?> byteBufferConstr = null;
```

Comment:
Eeh. What does native image says about this? It would be more native image friendly to use `MethodHandle`.

Reply:
The problematic is the fact that `DirectByteBuffer` is private.